backend/src/Application.ts

```diff
@@ -203,6 +203,31 @@ class ElectronApp extends EventEmitter {
       }
     })
   }
+
+  showWindow() {
+    if (this.window && this.tray) {
```

Comment:
What does this do exactly?

Reply:
It just shows the window in the correct location. It needs to check for the presence of this.window and this.tray to make typescript happy. If those vars were initialized a little different it says that the wouldn't have to be optional params.